AnimView/th.cpp

```diff
@@ -60,7 +60,7 @@ class ChunkRenderer {
 
   unsigned char* takeData() {
     unsigned char* buffer = m_data;
-    m_data = 0;
+    m_data = nullptr;
     return buffer;
```

Comment:
looks like a `std::move`, no idea if that is feasible in its environment. Different patch at least.

Reply:
This would be a case for std::move if we weren't working with raw pointers. As is the reference is switched without copying data already.

Now maybe we shouldn't be working with raw pointers...